apps/journeys/src/components/Conductor/NavigationButton/NavigationButton.tsx

```diff
@@ -100,7 +100,9 @@ export function NavigationButton({
 
   return (
     <NavigationContainer
-      data-testid={`${variant}NavContainer`}
+      data-testid={`NavigationContainer${variant
```

Comment:
Lets do this for all of the test ids in this component
```suggestion
      data-testid={`ContainerNavigationContainer${variant
```

Reply:
my bad, should be `ConductorNavigationContainer`